content/en/docs/concepts/extend-kubernetes/api-extension/declarative-validation.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Declarative Validation Concepts
```

Comment:
Couple of options here
```diff
-title: Declarative Validation Concepts
+title: Declarative API Validation
```

or
```diff
-title: Declarative Validation Concepts
+title: API Schemas & Validation
```

and add one or two sentences to say that Kubernetes validates writes against the schema. Which I know is obvious, but it's actually a better fit to have a page about schema validation in general than about one particular implementation of that.

Reply:
I have gone with the suggested title - `Declarative API Validation` for now